packages/browser/src/Component/Widget/Server/SPARQL/Service/SparqlEndpointService.js

```diff
@@ -16,13 +16,13 @@ export class SparqlEndpointService extends RequestService {
   constructor(configSparql) {
     super();
 
-    if (!!configSparql && !!configSparql.url && !!configSparql.url_parameters) {
-      // Wget "http://localhost:9999/strabon/Query?handle=download&query=%0ASELECT+*%0AWHERE+%7B+%0A%09%3Fs+%3Fp+%3Fo%09%0A%7D%0A&format=SPARQL/JSON&view=HTML"
-      this.url = configSparql.url;
-      this.url_parameters = configSparql.url_parameters;
-    } else {
+    if (!configSparql || !configSparql.url || !configSparql.url_parameters) {
```

Comment:
here is what should be in assets/config/widget/server/strabon.json (or assets/config/server/strabon.json after server folder migration)

Reply:
configSparql should be something like configStrabon

Reply:
added to assets/config/widget/server/sparql_server.json